src/teleop/jetson/arm_trajectory_server.py

```diff
@@ -122,8 +158,25 @@ def execute_cb(self, goal: FollowJointTrajectoryGoal) -> None:
                 self._feedback.actual.positions = joint_states.position
                 self._feedback.actual.velocities = joint_states.velocity
                 self._feedback.actual.time_from_start = rospy.Time.from_sec(time.time()) - time_start
+            self._feedback.error.positions = [
+                desired - actual
+                for desired, actual in zip(self._feedback.desired.positions, self._feedback.actual.positions)
+            ]
+            self._feedback.error.velocities = [
+                desired - actual
+                for desired, actual in zip(self._feedback.desired.velocities, self._feedback.actual.velocities)
+            ]
             self._as.publish_feedback(self._feedback)
             # ---------------------------------------
+            # Abort upon exceeding error threshold
+            error, error_msg = error_threshold_exceeded(self._feedback)
+            if error:
```

Comment:
Do you need to rospy.logerr() the error_msg? Or does it just get output by moveit?

Reply:
Just tested it, it gets automatically outputted along with the error code.